examples/sndcardparser/Android.bp

```diff
@@ -0,0 +1,7 @@
+cc_library {
+    name: "libsndcardparser",
```

Comment:
If this is called libsndcardparser, users will get conflicts when they define their own libsndcardparser, won't they? Why don't you name it libsndcardparser_example?

Reply:
done